composer.json

```diff
@@ -15,6 +15,10 @@
         {
             "name": "Sören Rohweder (monofone)",
             "email": "s.rohweder@blage.net"
+        },
```

Comment:
I'm sorry but there a several people with a lot more contributions that are not mentioned here. Please  be so kind and remove you as an author. Thx